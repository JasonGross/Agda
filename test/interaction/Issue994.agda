-- {-# OPTIONS -v interaction:50 #-}

module _ where

x : Set → Set
x = {!λ x → x!}

-- "refine" (C-c C-r) should behave the same as "give" here
-- Old, bad result:
-- x = λ x₁ → x₁
-- New, expected result:
-- x = λ x → x
