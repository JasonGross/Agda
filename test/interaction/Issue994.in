top_command (cmd_load currentFile [])
IOTCM currentFile None Indirect (Cmd_refine 0 (Range [Interval (Pn Nothing 20 2 7)  (Pn Nothing 27 2 14)]) "λ x → x")
