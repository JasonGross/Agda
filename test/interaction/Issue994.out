Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue994 (Issue994.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue994.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set â†’ Set\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-give-action 0 'no-paren)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 